<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddWorker</class>
 <widget class="QDialog" name="AddWorker">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>180</width>
    <height>210</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="phoneNumberLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>name</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="familyEditLine">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>161</width>
     <height>29</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="nameEditLine">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>161</width>
     <height>29</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="otchestvoLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>otchestvo</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="otchestvoEditLine">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>161</width>
     <height>29</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="familyLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>family</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addWorkerButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>161</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Add record</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
